libraries/core-react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@equinor/eds-core-react",
-  "version": "0.8.0",
+  "version": "0.8.2",
```

Comment:
Why not 0.8.1 here? 🤔

Reply:
Because we burned it 😂  Ref. issues with the non-working package

Reply:
Had to unpublish 0.8.1 from npm, and because of security npm doesn't allow you to republish with the same number. :)

Reply:
Ahaa that's right 🤣